bdd/features/e2e/E2E-014-python.feature

```diff
@@ -0,0 +1,118 @@
+Feature: Test our shiny new Python runner
+
+    @python
+    Scenario: E2E-014 TC-001 Run simple python sequence with input and output
```

Comment:
Just a thought: I think these tests could be run cross-language with some env vars pointing to the right directory with files.

Reply:
This would be nice, but I think it's out of scope here. Probably we could create an issue for this.